cmd/minikube/cmd/start.go

```diff
@@ -820,8 +820,12 @@ func validateKubernetesVersions(old *cfg.Config) (string, bool) {
 
 	if nvs.LT(ovs) {
 		nv = version.VersionPrefix + ovs.String()
-		out.ErrT(out.Conflict, "Kubernetes downgrade is not supported, will continue to use {{.version}}", out.V{"version": nv})
-		return nv, isUpgrade
+		exit.WithCodeT(exit.Config, `Error: You have selected version {{.new}}, but your existing cluster for profile "{{.profile}}" is running version {{.old}}. Non-destructive downgrades are not supported, but you can proceed by performing one of the following options:
+
+* Run "minikube start [-p {{.profile}}] --kubernetes-version={{.old}}" or newer to continue using this cluster
+* Run "minikube start -p <new name> --kubernetes-version={{.new}}" to start a new cluster with a different name
+* Run "minikube delete [-p {{.profile}}]" to delete the existing cluster first, and "minikube start [-p {{.profile}}] --kubernetes-version={{.new}}" to start a new cluster under the same name`, out.V{"new": nvs, "old": ovs, "profile": cfg.GetMachineName()})
```

Comment:
Great work! My apologies for not giving you the right advice up front, but after trying it out, I have some suggestions on improving the wording. For the easiest scanning of text, I feel that the beginning of the message should start with the choice outcome, followed with the steps the user must take.

```
* Recreate the cluster using v<old>: Run "minikube delete<profileFlag>", then "minikube ...
* Create a second cluster with v<old>: Run ...
* Reuse the existing cluster with v<new>: Run ...
```

What do you think?

Reply:
Yeah sure, that makes more sense